packages/web-components/src/components/cbp-form-field/cbp-form-field.stories.tsx

```diff
@@ -77,6 +150,79 @@ Textarea.args = {
 };
 
 
+
+
+function generateRadios(radios) {
+  const html = radios.map(({ label, name, value, checked, disabled }) => {
```

Comment:
Context isn't being passed to the radio buttons, same issue as with checklist sample